<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>createWindow</class>
 <widget class="QMainWindow" name="createWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>420</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Windows Manager - Create Group</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:rgb(58, 58, 58);
color:rgb(227, 227, 227)</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="createBox">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>100</y>
      <width>1241</width>
      <height>281</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lucida Sans</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QComboBox" name="selectBox">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>40</y>
       <width>471</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Lucida Sans</family>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color:rgb(52, 52, 52);
background-color:rgb(144, 144, 144)</string>
     </property>
    </widget>
    <widget class="QListWidget" name="instList">
     <property name="geometry">
      <rect>
       <x>840</x>
       <y>10</y>
       <width>341</width>
       <height>221</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Lucida Sans</family>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgb(91, 91, 91)</string>
     </property>
    </widget>
    <widget class="QPushButton" name="createGrpBtn">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>170</y>
       <width>471</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Lucida Sans</family>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color:rgb(52, 52, 52);
background-color:rgb(59, 218, 218)</string>
     </property>
     <property name="text">
      <string>Create Autoplay Group</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>840</x>
       <y>240</y>
       <width>341</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Lucida Sans</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Click on a Path to Remove it</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="colorButton">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>40</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lucida Sans</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(52, 52, 52);
background-color:rgb(59, 218, 218)</string>
    </property>
    <property name="text">
     <string>Change Symbol Colour</string>
    </property>
   </widget>
   <widget class="QLabel" name="colorLabel">
    <property name="geometry">
     <rect>
      <x>1170</x>
      <y>30</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lucida Sans</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(218, 218, 218);
margin:5px;
border:2px solid rgb(0, 0, 0)
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lucida Sans</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Group Name:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="nameEdit">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>40</y>
      <width>511</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lucida Sans</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(91, 91, 91)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>40</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lucida Sans</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Delay (s):</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="delayBox">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>50</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lucida Sans</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(52, 52, 52);
background-color:rgb(144, 144, 144)</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="createAction">
   <property name="text">
    <string>Create New Instance</string>
   </property>
  </action>
  <action name="openAction">
   <property name="text">
    <string>Open Existing Instances</string>
   </property>
  </action>
  <action name="quitAction">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="configAction">
   <property name="text">
    <string>Config Menu</string>
   </property>
  </action>
  <action name="locationAction">
   <property name="text">
    <string>Saved .exe Locations</string>
   </property>
  </action>
  <action name="autoplayAction">
   <property name="text">
    <string>Autoplay Speed</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
